swift/Sources/PiranhaKit/CleanupStaleFlags/StaleFlagCleaner.swift

```diff
@@ -714,6 +751,8 @@ class XPFlagCleaner: SyntaxRewriter {
             }
 
             if let rhs = binding.initializer {
+                updateScope(withIdentifier: binding.pattern.description.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines),
```

Comment:
the comment on Strings will help get rid of passing description and instead can probably operate on ExprSyntax.